### ISSUE TITLE
Feedback Ex 5 #12

### ISSUE THREAD

**Issue by kristianolesenlarsen** [Kristianuruplarsen/sdseml_online_exercises#33]
@Jbluni 

**The warmup**
This is a good attempt at a `Clock` class but there are some issues. First, your clock can have an hour > the maximum hour on the clock. This is because your `tick` method does not appropriately reset the hour when the time passes 0. Your `__eq__` method is missing, consider looking at [the solution](https://github.com/abjer/sds_eml_2020/blob/master/exercises/exercise_4_solution.pdf) to see my implementation.

**The exercises**
All of your code looks good, and the figures look good. About the R problem: is this happening in a completely fresh installation of R? And what OS are you on, as some of these cpp things are a bit OS sensitive?

**Comment by Jbluni** [comment]
Thanks for the feedback! I was not sure whether to reset the clock or not. But obviously it makes much more sense to reset.  I forgot the __eq__ method, but I looked at your solution. 

I don't know why it didn't worked, but now it is alright. I downloaded the newest R version. Haven't used it for a while.

I had a mistake. I defined v and therefore Tau wrong. But I fixed it now. I got the same figures at the end. Do I have to upload the new version?
